nuxeo-runtime/nuxeo-runtime-test/src/test/java/org/nuxeo/runtime/services/config/TestConfigurationService.java

```diff
@@ -104,17 +109,29 @@ public void testOverride() throws Exception {
     }
 
     @Test
+    @LogCaptureFeature.FilterOn(logLevel = "WARN")
     public void testCompatWarn() throws Exception {
         Framework.getProperties().setProperty("nuxeo.test.dummyStringProperty", "anotherDummyValue");
+        // Enable the dev mode to logs deprecated message as warning.
+        boolean devModeSet = Framework.isDevModeSet();
+        if (!devModeSet) {
+            Framework.getProperties().setProperty(Framework.NUXEO_DEV_SYSTEM_PROP, "true");
```

Comment:
I think it will be better to change the DeprecationLogger to make it log as WARN when we're currently testing too. Wdyt?

Reply:
if we add a warning logs automatically on DeprecationLogger as below :
`if (Framework.isDevModeSet() || Framework.isTestModeSet()) {` i think it will be hard to test the case of `INFO` is it true or i am missing something :) or we will end up by setting `org.nuxeo.runtime.testing` to false / true to get the expected behaviour

Reply:
If it's possible to do what you're saying about setting testing property to false, let's go for it.

Reply:
Is it necessary? because we're in test mode and condition should be evaluated to true without these lines